tldw_Server_API/tests/sandbox/test_admin_details_resource_usage.py

```
@@ -0,0 +1,60 @@
+        parts.append("sandbox")
+    os.environ["ROUTES_ENABLE"] = ",".join(parts)
+    from tldw_Server_API.app.main import app  # import after env is set
+    return TestClient(app)
+
+
+def _admin_user_dep():
+    from tldw_Server_API.app.core.AuthNZ.User_DB_Handling import User
+    return User(id=1, username="admin", roles=["admin"], is_admin=True)
+
+
+def test_admin_details_includes_resource_usage(monkeypatch) -> None:
```

Ruff (0.14.2)

Unused function argument: `monkeypatch`

(ARG001)